frontend/src/core/comments/components/Comment.test.js

```diff
@@ -16,19 +17,52 @@ describe('<Comment>', () => {
         id: 1,
     }
 
+    const DEFAULT_USER = {
+        username: 'Leslie_Knope',
+    };
+
+    const DEFAULT_CANREVIEW = {
+        canReview: true,
+    }
+
+    it('returns a string', () => {
+        const wrapper = shallow(<p>{ DEFAULT_COMMENT.content }</p>);
```

Comment:
This is testing the `<p>` tag from JSX. I assume you want to use the `Comment` component here instead?